src/en/charms-constraints.md

```diff
@@ -114,22 +126,13 @@ See [Creating a controller][controllers-creating] for more guidance.
     constraints from `--bootstrap-constraints` override any identical
     constraints from `--constraints`.
 
-For the localhost cloud, the following invocation will achieve a similar goal
-to the previous command (assuming that the LXD containers are using the
-'default' LXD profile):
+For the LXD cloud, the following invocation will place a **limit** of 2GiB of
+memory for each machine:
 
 ```bash
-lxc profile set default limits.memory 4GB
+juju bootstrap --constraints mem=2G localhost
```

Comment:
doesn't this need to be a --boostrap constraint? If we're just demonstrating a constraint on a workload let's show a deploy vs a bootstrap.

Reply:
But a workload constraint is **not** being demonstrated here.

Reply:
You compelled me to check `--constraints` behaviour. Turns out it does *not* set constraints for all models ('controller', 'default', and *all future models*), just 'controller' and 'default'. So I'll need to reword this entire section. IMO, `--constraints` should just be retired from the Juju code base as all it does more than `bootstrap-constraints` is add constraints to the 'default' model.